packages/gitbook/src/lib/references.tsx

```diff
@@ -41,8 +41,8 @@ export interface ResolvedContentRef {
     file?: RevisionFile;
     /** Page document resolved from the content ref */
     page?: RevisionPageDocument;
-    /** Resolved reusable content, if the ref points to reusable content on a revision. */
-    reusableContent?: RevisionReusableContent;
+    /** Resolved reusable content, if the ref points to reusable content on a revision. Also contains the space and revision used for resolution. */
+    reusableContent?: RevisionReusableContent & { space: string; revision: string };
```

Comment:
I'd recommend against extending interface and instead using 3 propa in an interface